libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -453,7 +452,7 @@ static int load_tf_model(TFModel *tf_model, const char *model_filename)
         av_freep(&sess_config);
         if (TF_GetCode(tf_model->status) != TF_OK) {
             TF_DeleteGraph(tf_model->graph);
-            TF_DeleteStatus(tf_model->status);
+            tf_model->graph = NULL;
```

Comment:
`tf_model->status` can be used in `ff_dnn_free_model_tf`, so it should not be free too early.
And the `tf_model->graph` should be set to NULL after deleting, to avoid double free in `ff_dnn_free_model_tf`.
The same reason for following change.